docs/pages/rmf-audio-capture_High-Level_TestSpec.md

```diff
@@ -0,0 +1,110 @@
+# RMF Audio Capture High Level Test Specification Documentation
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [Introduction](#introduction)
+- [Test Scenarios](#test-scenarios)
+  - [Check primary audio capture](#check-primary-audio-capture)
+  - [Check auxiliary audio capture](#check-auxiliary-audio-capture)
+  - [Check concurrent audio capture](#check-concurrent-audio-capture)
+
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` - Hardware Abstraction Layer, may include some common components
+- `UT` - Unit Test(s)
+- `SoC` - System on a Chip
+- `HAL` - Hardware Abstraction Layer
+- `API` - Application Programming Interface
+- `L2` - Level2 Testing
+- `L3` - Level3 Testing
+- `NA` - Not Applicable
+- `Y` - Yes
+
+
+## Introduction
+
+This document provides an overview of the testing requirements for the RMF audio capture module. It outlines the scope of testing, objectives, testing levels and approaches, specific test requirements, and expected deliverables.
+
+- `HAL` specification is available here: [rmf-audio-capture_halSpec.md](https://github.com/rdkcentral/rdk-halif-rmf_audio_capture/blob/main/docs/pages/rmf-audio-capture_halSpec.md)
+- `HAL` interface is available here: [rmfAudioCapture header](https://github.com/rdkcentral/rdk-halif-rmf_audio_capture/blob/main/include/rmfAudioCapture.h)
+
+
+## Test Scenarios
+
+|#|Test Functionality|Description|
+|-|------------------|-----------|
+|1|[Check primary audio capture](#check-primary-audio-capture)|Run a capture of primary audio for a while and verify delivery of data|
+|2|[Check auxiliary audio capture](#check-auxiliary-audio-capture)|Run a capture of auxiliary audio (on supported devices only) for a while and verify delivery of data|
+|3|[Check concurrent audio capture](#check-concurrent-audio-capture)|Run parallel captures of primary and auxiliary audio (on supported devices only) and verify delivery of data|
+
+### Check primary audio capture
+
+|Description|HAL APIs|L2|L3|Control plane requirements|
```

Comment:
i think one point missed we need to keep source & sink column 
https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/main/docs/pages/ds-video-device_High-Level_TestSpec.md